apps/cyberstorm-remix/app/p/packageListing.tsx

```diff
@@ -426,22 +420,34 @@ export default function PackageListing() {
                     </Await>
                   </Suspense>
                 </Drawer>
-                <Suspense fallback={<p>Loading...</p>}>
-                  <Await resolve={listingAndTeamPromise}>
-                    {(resolvedValue) => (
-                      <Actions
-                        team={resolvedValue[1]}
-                        listing={resolvedValue[0]}
-                        isLiked={isLiked}
-                        currentUser={currentUser}
-                        likeUpdateTrigger={fetchAndSetRatedPackages}
-                        requestConfig={config}
-                      />
-                    )}
-                  </Await>
-                </Suspense>
-
-                {ReportPackageButton}
+                <div className="package-listing__narrow-other-actions">
+                  <Suspense fallback={<p>Loading...</p>}>
+                    <Await resolve={listingAndTeamPromise}>
+                      {(resolvedValue) => (
+                        <Actions
+                          team={resolvedValue[1]}
+                          listing={resolvedValue[0]}
+                          isLiked={isLiked}
+                          currentUser={currentUser}
+                          likeUpdateTrigger={fetchAndSetRatedPackages}
+                          requestConfig={config}
+                        />
+                      )}
+                    </Await>
+                  </Suspense>
+                  <Suspense fallback={<ReportPackageButton />}>
+                    <Await resolve={listing}>
+                      {(resolvedValue) => (
+                        <ReportPackage
```

Comment:
After the changes in this PR, it would probably make sense to move this back into Actions. It was extracted from there just because my implementation wasn't very compatible with it.

Reply:
What I tried to say was that [before my PRs](https://github.com/thunderstore-io/thunderstore-ui/pull/1582/files#diff-caa284e2f1e08c53540f6f4d8ab5a0ba0b5559beed976f69ba6a0d1a21b74705L1169), PackageReportButton, as well as `<div className="package-listing-sidebar__actions">` were contained in the Actions component, which made sense as they're somewhat a logical unit. I only extracted them because my implementation (custom hook) required it. If the custom hook is dropped, I'd say it makes sense to put them back in. Not a blocker but would make things cleaner IMO.